hawkbit-rest/hawkbit-mgmt-api/pom.xml

```diff
@@ -24,6 +24,10 @@
          <groupId>org.springframework.hateoas</groupId>
          <artifactId>spring-hateoas</artifactId>
       </dependency>
+      <dependency>
+         <groupId>org.springframework</groupId>
```

Comment:
I don't think this dependency is required for the API or did I overlooked something?

Reply:
Removed it.